src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/product.yml

```diff
@@ -27,3 +27,12 @@ admin_products_v2_edit:
       id_product: productId
   requirements:
     productId: \d+
+
+admin_products_v2_download_virtual_product_file:
+    path: /virtual-product-file/{virtualProductFileId}
+    methods: [GET, POST, PATCH]
```

Comment:
@jolelievre, are you sure we need all these methods? shouldn't just "GET" be available?

Reply:
Not at all it's just a bad copy paste 😅